packages/api/src/functions/dbAuth/shared.ts

```diff
@@ -44,9 +50,9 @@ export const getSession = (text?: string) => {
 // Convenience function to get session, decrypt, and return session data all
 // at once. Accepts the `event` argument from a Lambda function call.
 export const dbAuthSession = (event: APIGatewayProxyEvent) => {
-  if (event.headers.cookie) {
+  if (event.headers.cookie || event.headers.Cookie) {
```

Comment:
```suggestion
  if (extractCookie(event)) {
```

This would work, right? I feel like there should only be one place that needs to know how to get the cookie, and it's in `extractCookie()`!